tools/issue-labeler/main.go

```diff
@@ -1,23 +1,40 @@
 package main
 
 import (
+	"flag"
 	"fmt"
 	"os"
+	"sort"
+	"strings"
 
 	"github.com/golang/glog"
 	"gopkg.in/yaml.v2"
 )
 
+var flagBackfill = flag.String("backfill-date", "", "run in backfill mode to apply labels to issues filed after given date")
+var flagDryRun = flag.Bool("backfill-dry-run", false, "when combined with backfill-date, perform a dry run of backfill mode")
+
 func main() {
-	issueBody := os.Getenv("ISSUE_BODY")
+	flag.Parse()
+
 	file, err := os.ReadFile("enrolled_teams.yaml")
 	if err != nil {
 		glog.Exitf("Error reading enrolled teams yaml: %v", err)
 	}
 	enrolledTeams := make(map[string][]string)
-	err = yaml.Unmarshal(file, &enrolledTeams)
-	if err != nil {
+	if err := yaml.Unmarshal(file, &enrolledTeams); err != nil {
 		glog.Exitf("Error unmarshalling enrolled teams yaml: %v", err)
 	}
-	fmt.Println(labels(issueBody, enrolledTeams))
+
+	if *flagBackfill == "" {
+		issueBody := os.Getenv("ISSUE_BODY")
```

Comment:
Since we're using flags now, this may be better suited as a flag as well. Can we pass the issue body as a flag in the GitHub Action?

Part of the reason I ask is that I think we should write this in a way where a GitHub Action can be used to call either version of the labeler (new issue or backfill).

Reply:
I think passing it as an environment variable makes more sense. I'm not sure what would happen to the formatting with such a long flag.

Reply:
Ah ok, I'm not sure about the flag either, but seems reasonable to keep it an env var for that reason